package.json

```diff
@@ -25,21 +25,24 @@
   },
   "dependencies": {},
   "devDependencies": {
```

Comment:
quite a few dependabot PRs popped up after I merged `3.0.0-beta`, hopefully those get knocked out as well with the fresh `npm i` - I saw `lodash` got bumped to 4.17.15 in the lockfile

Reply:
I'll do those separately. Want to get something working in to make sure we don't regressions.